development/slider-carousel-app/pages/slider-carousel/index.mdx

```diff
@@ -0,0 +1,120 @@
+---
+HEAD OBJECT
+title: "ReuseJS slider-carousel"
+author: "Md Amir Sohail"
+---
+import SliderWithButton from "../../components/slider-carousel/Example/SliderWithButton/SliderWithButton.tsx"
+import SimpleSlider from "../../components/slider-carousel/Example/SimpleSlider/SimpleSlider.tsx"
+import HeadlessSlider from "../../components/slider-carousel/Example/HeadlessSliderExample/HeadlessSlider.tsx"
+import SliderWithProgress from "../../components/slider-carousel/Example/SliderWithProgress/SliderWithProgress.tsx"
+
+## Table of contents
+
+## Introduction
+***Take your website to the next level with ReuseSliderCarousel. Effortlessly create stunning image carousels and captivating slideshows. Customize slide intervals, enable looping, and add intuitive button controls. Engage your visitors with seamless transitions and eye-catching animations. Showcase your content in style and leave a lasting impression.***
+
+## HeadlessSliderCarousel Component Documentation
+The HeadlessSliderCarousel component is a customizable carousel/slider component for displaying a set of slides. It provides automatic scrolling functionality and is designed to be headless, meaning it handles the core functionality of the slider while allowing you to control the presentation and styling.
+## HeadlessSliderCarousel props
+- **slides** (required) :  An array of ReactNode elements representing the individual slides to be displayed in the carousel.
+- **dependency** (required) : A dependency value that triggers the carousel to resume scrolling when it changes. This can be useful when you want to update the carousel based on external changes, such as data updates.
+
+**Node :**
```

Comment:
What is Node????

Reply:
ReactNode is And HTML Element , Replaced With React Elemennt